### DISCUSSION TITLE
Question:  Does this work with Home Assistant Docker?

### DISCUSSION THREAD

**Discussion by mew0717** [jcwillox/hass-auto-backup#33] [Q&A] [answered]
Or does this require supervisor and HassOS?

**Comment by jcwillox** [comment, marked as answer]
Auto Backup uses the supervisor to create snapshots, it doesn't create them itself. The goal of Auto Backup is to provide an easy way to create and automatically delete snapshots using automations/service calls. 

Basically, Auto Backup will only work if the supervisor is present, so HassOS will work, or if your using the generic linux installer, but standalone Home Assistant instances won't.

**Comment by kaijk** [comment]
I got confused where the *[download_path](https://jcwillox.github.io/hass-auto-backup/services/#download-path)* discussion references running in Docker and directory paths being relative to the container.  Thank you for the clarification.
